pkg/workflows/wasm/host/wasm_test.go

```diff
@@ -459,28 +476,92 @@ func Test_Compute_Fetch(t *testing.T) {
 			assert.Equal(t, expectedEntries[i].Log.Message, logs.AllUntimed()[i].Entry.Message)
 		}
 	})
+
+	t.Run("OK_context_propagation", func(t *testing.T) {
```

Comment:
should test that the context also cancels a function call by making a fetch that waits on ctx.Done()

Reply:
👍🏼  thanks! added!